### ISSUE TITLE
Adding Assignment Section for WordPress Login, Register and Lost Password pages.

### ISSUE THREAD

**Issue by ilkayivrendi** [gantry/gantry5#2009]
I believe this will be one easy and  useful feature if adding one more section for word WordPress Core login, register and lost password assignments with chosen outline. I know I can add custom login and reset password input fields with particles or with short codes but it is so much time consuming for simple step and in so many cases plugins needs to be used for changing wp-login.php link.

 So many people using extra plugins to change this features for comparable with their themes and hide the WordPress original login and register sections for security reasons.  

My opinion is using one framework for all needs what one theme need. BuddyPress and bbPress is easy to compatible with Gantry but some functions links still needs to be customized with WordPress core and if we can manage this simple pages with Gantry It can reach to more user... 

People dont want to use more plugins and spend time for it if they already have one strong framework which can handle everything for them with one click and all they can focus is design and UI/UX. 

One framework for everything... 

Thank You.

**Comment by mahagr** [comment]
Do you have an example/instruction somewhere on how to customize these?

**Comment by newkind** [comment]
To be honest these pages like login or register aren't standard WP pages and they require some more coding skills. I consider them system pages and I'm not sure if it's a good idea to add them to assignments. Plus we would have to introduce our own conditional checks as WP doesn't provide any for these pages.

**Comment by ilkayivrendi** [comment]
@newkind Yes it requires more codding skills. That is why I open this topic here. I solve the problem with one plugin named Theme My Login. This plugin basically changing wp_login_url() to my specific url and after login success than redirects user to where you wish to. In BuddyPress Widged Login link was redirecting me to my front end login but menu login link was redirecting me to admin login page.  You can check how it works now from my website now http://ivrenditech/login/ after you login you will see that login and register menu is disappears  and profile and sub-menus are showing.

@mahagr I suppose for these changes you need to find assigments.yaml and assigments.html.twig files from plugin and replace with your own functions with custom/engine/ directory/ (Directory address may be wrong. I cant reach the files now to check correct path)

**Comment by mahagr** [comment]
I don't think you should be able to assign an outline for the login page, but maybe there could be system outline which gets used with the login page.

**Comment by ilkayivrendi** [comment]
Yes. I believe that can also work. Currently I just focused on front end and particle system which general users may need like prizing box, charts ext. I didn't had time for back-end and function binding.  I gained twig-timber knowledge a bit while I was playing around with gantry but it is not strong enough to make those changes.
